readthedocs/proxito/views/mixins.py

```diff
@@ -176,6 +177,12 @@ def get_version_from_host(self, request, version_slug):
             version_slug = request.host_version_slug
         return version_slug
 
+    def _spam_response(self, project):
+        if 'readthedocsext.spamfighting' in settings.INSTALLED_APPS:
+            from readthedocsext.spamfighting.utils import serve_docs
+            if not serve_docs(project):
```

Comment:
What about prefixing all these functions with `deny_`? eg. `deny_serving_docs`, `deny_showing_dashboard`, `deny_showing_ads`

Reply:
or prefix with `dont_` as I did in the admin filters?

Reply:
@humitos This might be good to clear up before merge too.

I don't have opinions here, but will throw one more general suggestion in: using auxiliary verbs in the member naming.

`deny_serve_docs` - "deny" here is ambiguous, and could be pasred as a question ("is serve docs denied?") or a statement ("block this request from serving docs").

Adding auxiliary verbs clarifies this. `is_serve_docs_denied` parses more immediately as a function that checks if serve docs is allowed. Also neat: it implies a boolean operation and suggests a yes/no answer in return.

It's wordier, but it's clear.

It might save the next person time or confusion while scanning through the code as well. All these suggestions solve the overlap with the underlying `serve_docs` though, so any sound fine.

Reply:
@agjohnson sold! I'll adapt this code to use that naming.